LocalSettings.php

```diff
@@ -4568,14 +4568,7 @@
 					'PasswordNotInCommonList' => [ 'value' => true, 'suggestChangeOnLogin' => true ],
 				],
 			],
-			'checks' => [
-				'MinimalPasswordLength' => 'PasswordPolicyChecks::checkMinimalPasswordLength',
-				'MinimumPasswordLengthToLogin' => 'PasswordPolicyChecks::checkMinimumPasswordLengthToLogin',
-				'PasswordCannotBeSubstringInUsername' => 'PasswordPolicyChecks::checkPasswordCannotBeSubstringInUsername',
-				'PasswordCannotMatchDefaults' => 'PasswordPolicyChecks::checkPasswordCannotMatchDefaults',
-				'MaximalPasswordLength' => 'PasswordPolicyChecks::checkMaximalPasswordLength',
-				'PasswordNotInCommonList' => 'PasswordPolicyChecks::checkPasswordNotInCommonList',
-			],
+			// Checks is set near bottom due to requring different config under 1.43+
```

Comment:
There is support for using version tags so you can do '+1.42' => [ 'checks' => for 1.42 and the same for 1.43 here